invenio_cli/commands/requirements.py

```diff
@@ -113,7 +114,7 @@ def check_docker_version(cls, major, minor=-1, patch=-1, exact=False):
         # Output comes in the form of
         # 'Docker version 19.03.13, build 4484c46d9d\n'
         try:
-            result = run_cmd(["docker", "--version"])
+            result = run_cmd(["docker", "version"])
```

Comment:
We list in the [system requirements docs](https://inveniordm.docs.cern.ch/install/requirements/) `Docker 20.10.10+` since [we migrated the Docker images](https://github.com/inveniosoftware/docs-invenio-rdm/pull/513). 

As you said, there might be dangers with just upgrading Docker and running commands on existing containers/setups, but I think we never advertised `docker-compose` as a production setup so it's only going to be an issue during development.

I would say this is a minor bump, given how close we are to the June EOL of Docker v1.

Reply:
@slint @fenekku Given the impact of this change, even if I agree that is not breaking change for the public API per see, I am leaning towards a major.... Even if it doesn't fall in this category https://semver.org/#what-if-i-inadvertently-alter-the-public-api-in-a-way-that-is-not-compliant-with-the-version-number-change-ie-the-code-incorrectly-introduces-a-major-breaking-change-in-a-patch-release I like the answer given there
```
Remember, Semantic Versioning is all about conveying meaning by how the version number changes. If these changes are important to your users, use the version number to inform them.
```
WDYT?

Reply:
Yeah it's not super clear cut and semantic versioning creates or struggles with weird marketing expectations (From that link: "Semantic Versioning is all about conveying meaning by how the version number changes. If these changes are important to your users, use the version number to inform them." -- that's a lot of leeway :sweat: ).

I lean for a minor bump per the "it's the dependency that changed and not the project's API" argument, as well as @slint's note on the pre-existing requirement change that didn't lead to a major bump. I think of docker-compose as a regualr python dependency/python version now and how there are typically no major version bump in a project as an older python version/package version is deprecated in favor of a new one.